sos/report/plugins/openstack_cinder.py

```diff
@@ -48,6 +49,71 @@ def setup(self):
                 "cinder-manage " + cinder_config + " db version",
                 suggest_filename="cinder_db_version"
             )
+            self.add_cmd_output(
+                f"cinder-manage {cinder_config} backup list",
+                suggest_filename="cinder-manage_backup_list"
+            )
+            self.add_cmd_output(
+                f"cinder-manage {cinder_config} config list",
+                suggest_filename="cinder-manage_config_list"
+            )
+            self.add_cmd_output(
+                f"cinder-manage {cinder_config} host list",
+                suggest_filename="cinder-manage_host_list"
+            )
+            self.add_cmd_output(
+                f"cinder-status {cinder_config} upgrade check",
+                suggest_filename="cinder-status_upgrade_check"
+            )
+
+            vars_all = [p in os.environ for p in [
+                        'OS_USERNAME', 'OS_PASSWORD']]
+
+            vars_any = [p in os.environ for p in [
+                        'OS_TENANT_NAME', 'OS_PROJECT_NAME']]
+
+            if not (all(vars_all) and any(vars_any)):
+                self.soslog.warning("Not all environment variables set. "
+                                    "Source the environment file for the user "
+                                    "intended to connect to the OpenStack "
+                                    "environment.")
+            else:
+                list_cmds = [
+                    "backend pool",
+                    "group type",
+                    "message",
+                    "qos",
+                    "service",
+                    "type",
+                ]
+
+                for cmd in list_cmds:
+                    self.add_cmd_output(f"openstack volume {cmd} list")
+
+                list_cmds_projects = [
+                    "backup",
+                    "group",
+                    "group snapshot",
+                    "snapshot",
+                    "transfer request",
+                ]
+
+                for cmd in list_cmds_projects:
+                    self.add_cmd_output(
+                        f"openstack volume {cmd} list --all-projects"
+                    )
+
+                self.add_cmd_output("openstack volume list --all-projects")
```

Comment:
You might add it to the previous `for cmd in ..` loop - bit more concise code, bit less readable, I guess.

Leaving on you the decision, both is good for me.